### ISSUE TITLE
Polymer Init Element Not Using bower_components

### ISSUE THREAD

**Issue by andrewweaver** [Polymer/polymer-cli#517]
### Description
The generator for an element project (`polymer init`) doesn't support bower_components

### Versions & Environment
- Polymer CLI: 0.17.0
- node: 7.1.0
- Operating System: OSX Sierra

#### Steps to Reproduce
1. Create new directory and `cd` into it
2. Run `polymer init`
3. Select element option and input name, description, etc.
4. Serve the element using `polymer serve` or `python -m SimpleHTTPServer 8000`
5. Navigate to page in browser (I have Chrome 54)

#### Expected Results
Should show standard polymer documentation page.

#### Actual Results
Unable to load resources, see image below:
![screen shot 2016-12-15 at 3 28 49 pm](https://cloud.githubusercontent.com/assets/16942714/21240897/48c40ca6-c2db-11e6-8a47-6c755bb3ae3e.png)

I believe its simply the HTML imports not linking appropriately to the `bower_components` folder.

**Comment by FredKSchott** [comment]
Hmm, I can't reproduce this and haven't heard this from anyone else. Did `bower install` run correctly when you created the element?

**Comment by andrewweaver** [comment]
I tested again last night on my other Mac as well as my desktop PC. Same issue on all even after manually running `bower install` and `bower update`. I attempted to change the import dependencies in the `index.html` and the documentation page started showing through.
<img width="920" alt="screen shot 2016-12-16 at 9 18 52 am" src="https://cloud.githubusercontent.com/assets/16942714/21265725/b1ef8198-c370-11e6-83a1-1ee172fe3eed.png">
